src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SamlNameIdentifierClaimResource.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Serialization;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    internal class SamlNameIdentifierClaimResource
+    {
```

Comment:
You dropped the DataContract attribute. Same thing applies about namespace and field names.